railties/lib/rails/generators/rails/app/templates/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "<%= app_name %>",
+  "private": true,
+  "dependencies": {
+    "jquery": "~3.1.0",
+    "jquery-ujs": "~1.2.2"
+  }
```

Comment:
I think we will be able to remove jquery from here and change `jquery-ujs` to `rails-ujs` after of having https://github.com/rails/rails/issues/25208 finished